setup.py

```diff
@@ -21,8 +19,21 @@
         "Programming Language :: Python :: 3.6",
```

Comment:
I know you haven't changed these in this PR, but why are there versions of python here that are incompatible with our app?

Reply:
Good catch, I will fix that on this branch.

Reply:
See 883b3c5